modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js

```diff
@@ -121,6 +124,15 @@ const ResultsBar = ({
 					</ClayLink>
 				</ManagementToolbar.ResultsBarItem>
 			</ManagementToolbar.ResultsBar>
+
+			{title ? (
```

Comment:
Should we add  `showTitle` check here as well? We could also add FF check, to match three checks in SSR.

Reply:
What I meant with "showTitle" is to _visually_ show it or not, that's why I use it to toggle the `sr-only` class (like I do in the java class here: https://github.com/liferay-frontend/liferay-portal/pull/3779/files#diff-693ec17285b9e3a9bd29a88403e0b1a280ca0c79d47868883f93ddaa8d19d47eR1357-R1359).

Maybe the name is a bit confusing and it should be `visuallyShowTitle`, instead? I wanted to take into account cases where we do not want to show the title in the UI, but still needs to be part of the accessibility tree.

If you find it too confusing I can just remove that extra attribute from the PR, it is not needed for the story :sweat_smile: 

I should check the FF too, that's right. I'll add it :smile:

Reply:
Oh wait, yeah, I am misunderstanding your intent. Other `show` are "toggle feature" attributes, so this would be misleading :thinking:

Reply:
What we could do is:
1. Render `searchResultsTitle` in UI, when it is set. Screen readers will catch this.
2. When `searchResultsTitle` is not set, render sr-only title, hardcoded to "Search Results".

What do you think?

Reply:
> I don't mind an explicit flag to toggle a feature. What I don't like is show* naming pattern, as it results in the awkward isShow* getters. In new components, I prefer naming such attributes *Enabled. But, show in this PR is consistent with other existing show attributes in MT, so we might as well keep it 🤷‍♂️

Ahhh alright, I dislike them those methods too :laughing: 

> Oh wait, yeah, I am misunderstanding your intent. Other show attributes are "toggle feature" attributes, so this would be misleading 🤔

Yep, that is what I thought

> What we could do is:
>    1. Render searchResultsTitle in UI, when it is set. Screen readers will catch this.
>    2. When searchResultsTitle is not set, render sr-only title, hardcoded to "Search Results".
> What do you think?

I would be a great idea, but they want to show the "search results" title by default, so we would need to add this to each management bar usage.

I think I overcomplicated this with the extra attribute, and they (PMs) did not request that, sooo... what if I just remove it :sweat_smile:

Reply:
Yeah, I think it's really not necessary. And `sr-only` part does seem to me needed, if the title will always be there.